pycds/climate_baseline_helpers.py

```diff
@@ -0,0 +1,169 @@
+from __future__ import print_function
+
+"""Tools for loading climate baseline data into database from flat files.
+"""
+
+import struct
+import datetime
+from calendar import monthrange
+from pycds import Network, History, Variable, DerivedValue
+
+pcic_climate_variable_network_name = 'PCIC Climate Variables'
+
+
+class InvalidVariableName(Exception):
+    pass
+
+
+def get_or_create_pcic_climate_variables_network(session):
+    """Get or, if it does not exist, create the synthetic network for derived variables
+
+    Args:
+        session (...): SQLAlchemy session for accessing the database
+
+    Returns:
+        Network object for synthetic network for derived variables
+    """
+
+    network = session.query(Network).filter(Network.name == pcic_climate_variable_network_name).first()
```

Comment:
Maybe ``pcic_climate_variable_network_name`` should be an argument to the function with the default being the constant defined above? Otherwise this function is just a subroutine with some magic values hidden inside.